client/client.go

```diff
@@ -29,9 +29,10 @@ import (
 func init() {
 	data.SetDefaultExpiryTimes(
 		map[string]int{
-			"root":     3650,
```

Comment:
Agreed, we could potentially put them in `const.go` to ensure they're shared consistently